content/src/content/jcr_root/apps/core/wcm/sandbox/components/tabs/v1/tabs/clientlibs/editor/js/tabs.js

```diff
@@ -0,0 +1,73 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2018 Adobe Systems Incorporated
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+/* global jQuery */
+(function($) {
+    "use strict";
+
+    var dialogContentSelector = ".cmp-tabs__editor";
+    var childreneditorSelector = ".cmp-childreneditor";
+    var childreneditorIconTitleSelector = ".cmp-childreneditor__item-icon [title]";
+    var childreneditorItemSelector = ".cmp-childreneditor__item-title";
+    var activeItemSelector = "input[name='./activeItem']";
+    var activeSelectSelector = "coral-select.activeSelect";
+
+    $(document).on("dialog-loaded", function(e) {
+        var $dialog = e.dialog;
+        var $dialogContent = $dialog.find(dialogContentSelector);
+        var dialogContent = $dialogContent.length > 0 ? $dialogContent[0] : undefined;
+
+        if (dialogContent) {
+            var $childrenEditor = $dialogContent.find(childreneditorSelector);
```

Comment:
no need for jQuery:
`dialogContent.querySelector(childrenEditorSelector)`

Reply:
Done in https://github.com/Adobe-Marketing-Cloud/aem-core-wcm-components/pull/317/commits/f1cdc202aed9bbb98c58c7bdd5c5b6ca7a017b4e